### PR TITLE
fit: Implement reusable send-email helper

### PR DESCRIPTION
**Pull request by patrickhag**
[187481255](https://www.pivotaltracker.com/story/show/187481255)
## Purpose

* To create reusable sendEmail fn that will be used to send all kinds of email

## Changes Made

* Added in `send-email` fn in helpers directory
* Added two modules that are `nodemailer` and `mailgen`

## Testing Instructions

* None

## Related Issues

* None

## Checklist

Please review the following checklist and make sure all tasks are complete before submitting:

- [x] Code follows the project's coding standards
- [x] Changes are covered by tests
- [x] Documentation is updated (if applicable)
- [x] All tests pass

### CODE REVIEW

File: src/helpers/send-email.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  name: string;
+}
+
+const { EMAIL, PASSWORD } = process.env;
+
+export const sendEmail = async (type: string, data: IData) => {
+  try {
+    const mailGenerator = new Mailgen({
+      theme: 'default',
+      product: {
```
Comment (JeanIrad) [inline src/helpers/send-email.ts:19]: that's awesome work

**Review by JeanIrad — APPROVED** [review]

**Review by hozayves — APPROVED** [review]

**Review by hozayves — CHANGES_REQUESTED** [review]
Hello Patrick, 
thanks for your incredible work but I want to tell you that this function is insufficient. what if I want to send verification mail I will need these parameters:  `type, name, link, email` and also there time we could need to send specific `message`

File: src/helpers/send-email.ts (hunk 1 above)
Comment (patrickhag) [reply src/helpers/send-email.ts:19]: thanks man

**Review by niyontwali — APPROVED** [review]
Well done

**Review by Mag-codes — APPROVED** [review]

**Review by P-Rwirangira — APPROVED** [review]

**Pull request merged**